android/app/build.gradle

```diff
@@ -197,13 +199,14 @@ android {
             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
             signingConfig signingConfigs.release
             matchingFallbacks = ['release']
+
         }
     }
 
     flavorDimensions "version"
     productFlavors {
         bt {
-            applicationId 'org.pathcheck.covidsafepathsBt'
```

Comment:
I think we were using this id to differentiate the GAEN app from the GPS one, and have both installed and the same time.
@johnschoeman @octohub We should confirm with id we are going to use before releasing versions to the Play Store because it can be changed afterwards.

Reply:
Can you suggest an alternative to this?

Reply:
If possible I would keep `org.pathcheck.covidsafepathsBt`

Reply:
Did you mean change back to org.pathcheck.covidsafepathsBt in packages.json?  I tried that and it did not install.

Reply:
I mean reverting the change here and in these 3 places
https://github.com/Path-Check/gaen-mobile/pull/71/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L13
https://github.com/Path-Check/gaen-mobile/pull/71/files#diff-3e828e9281e5dfb593f53c53b1cec1d2L112
https://github.com/Path-Check/gaen-mobile/pull/71/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L13

Did you get any error with the previous applicationId?

Reply:
Yes, that's why I changed them.

Reply:
Do you know what error it was? 
Is it related to Firebase or it also happens on `develop` branch?

Reply:
Oh I think the error is due to the uppercase letter
We can change the package name to `org.pathcheck.covidsafepaths.bt` in that case

Reply:
OK, i'll try that

Reply:
It only happened in the Firebase branch